Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -4711,6 +4280,12 @@ script:
       name: hostname
       required: false
       secret: false
+    - default: true
+      description: ''
+      isArray: true
+      name: endpoint
+      required: true
+      secret: false
```

Comment:
I believe this is the source of the validation error, as adding a required new argument can break the backward compatibility. Can you look into it and remove if it was by mistake?

Reply:
Removed this part. Did not touch this command, but it maybe was a "fix" by demisto-sdk format as I ran this before commiting